docs/guides/hybrid-models.md

```diff
@@ -1,20 +1,35 @@
 # Hybrid models
 
-FHE enables cloud applications to process private user data without running the risk of data leaks. Furthermore, deploying ML models in the cloud is advantageous as it eases model updates, allows to scale to large numbers of users by using large amounts of compute power, and protects model IP by keeping the model on a trusted server instead of the client device.
+This document explains how to use Concrete ML API to deploy hybrid models in Fully Homomorphic Encryption (FHE).
 
-However, not all applications can be easily converted to FHE computation and the computation cost of FHE may make a full conversion exceed latency requirements.
+## Introduction
 
-Hybrid models provide a balance between on-device deployment and cloud-based deployment. This approach entails executing parts of the model directly on the client side, while other parts are securely processed with FHE on the server side. Concrete ML facilitates the hybrid deployment of various neural network models, including MLP (multilayer perceptron), CNN (convolutional neural network), and Large Language Models.
+FHE allows cloud applications to process private user data securely, minimizing the risk of data leaks. Deploying machine learning (ML) models in the cloud offers several advantages:
+
+- Simplifies model updates.
+- Scales to large user bases by leveraging substantial compute power.
+- Protects model's Intellectual Property (IP) by keeping the model on a trusted server rather than on client devices.
+
+However, not all applications can be easily converted to FHE computation. The high computation cost of FHE might exceed latency requirements for full conversion.
+
+Hybrid models provide a balance between on-device deployment and cloud-based deployment. This approach involves:
+
+- Executing parts of the model on the client side.
+- Securely processing other parts with FHE on the server side.
+
+Concrete ML supports hybrid deployment for various neural network models, including Multilayer Perceptron (MLP), Convolutional Neural Network (CNN), and Large Language Models.
```

Comment:
to keep the sentence structure harmonious, you can add: and Large Language Models (LLM).